content/sigs/platform/index.adoc

```diff
@@ -67,13 +67,11 @@ ARM architecture support, adapting RedHat packaging to best practices, etc.)
 
 === Ongoing projects
 
-* Java 11 support in Jenkins (jep:211[])
 * Multi-architecture Docker images
 ** Enabling official images to run on Arm, IBM s390x, and other platforms
 * Rework of Windows installers
 ** New version of installers for 64bit platforms
 ** Removal of Java bundling
-** Creating an official link:https://chocolatey.org/packages/jenkins[Jenkins Chocolatey package]
```

Comment:
We still need to document it somewhere. Might be similar to the recent macOS patches. Cc @slide

Reply:
Agreed.  Restored in https://github.com/jenkins-infra/jenkins.io/pull/2665/commits/ab7e4d357f2dfbad69f2a6cf8b6c315c4be99d50 so that I can merge the rest of these changes.